### PR TITLE
Revamp: global style

### PR DESCRIPTION
**Pull request by Gregcop1**
Here comes a screenshot  (with a bug in the screenshot export on icons around the spider)
screencapture-localhost-3000-1477436595410

### CODE REVIEW

File: src/containers/Home/Home.js [hunk 1]
```diff
@@ -41,19 +68,22 @@ export default class Home extends Component {
...
+        <div id="incredible" className="incredible" style={{minHeight: this.state.windowHeight}}>
           <section className="container">
-            <img src={logoImage} alt="API Platform" width={300}/>
-            <p>The ultimate <a href="http://php.net" target="_blank">PHP</a> framework to build modern web APIs.<br /></p>
+            <h1><img src={logoImage} alt="API Platform" height={400}/></h1>
+            <p>The ultimate <a href="http://php.net" target="_blank">PHP</a> framework to build modern web APIs.</p>
 
             <Link className="btn btn-primary btn-lg" to="/docs/distribution/" role="button"><span className="fa fa-plus-circle"/> Get started</Link>
-            <a className="btn btn-primary btn-lg" href="https://demo.api-platform.com" role="button"><span className="fa fa-mouse-pointer" />Demo</a>
+            <Link className="btn btn-primary btn-lg" to="https://demo.api-platform.com" role="button"><span className="fa fa-mouse-pointer"/>Demo</Link>
```
Comment (dunglas) [inline src/containers/Home/Home.js]: Should be `<a>` tag because it's an external URL.

File: src/containers/Home/Home.js [hunk 2]
```diff
@@ -41,19 +68,22 @@ export default class Home extends Component {
...
           </section>
         </div>
 
-        <div id="features" className="slide">
+        <HeaderNav inline={true}/>
+
+        <div id="features" className="slide features">
           <section className="container">
-            <h1>Creating Linked Data <a href="https://en.wikipedia.org/wiki/Representational_state_transfer" target="_blank">REST</a> APIs has never been so easy</h1>
+            <h2>Creating Linked Data <a href="https://en.wikipedia.org/wiki/Representational_state_transfer" target="_blank">REST</a> APIs has never been so easy</h2>
```
Comment (dunglas) [inline src/containers/Home/Home.js:86]: Shouldn't it a `<h1>` tag (first title element of a `<section>`, IIRC the `<section>` element reset the titling level.

File: src/containers/Home/Home.js [hunk 3]
```diff
@@ -63,78 +93,54 @@ export default class Home extends Component {
...
               </div>
             </div>
           </section>
         </div>
 
-        <div id="architecture" className="slide">
+        <div id="architecture" className="architecture slide">
           <section className="container">
-            <h1>The web has changed!</h1>
+            <h2>The web has changed!</h2>
```
Comment (dunglas) [inline src/containers/Home/Home.js:112]: Same here, why changing to a `<h2>`?

File: src/containers/Home/Home.js [hunk 4]
```diff
@@ -63,78 +93,54 @@ export default class Home extends Component {
...
 
             <ul>
               <li>
-                <b>Javascript webapps (<a href="https://en.wikipedia.org/wiki/Single-page_application" target="_blank">SPA</a></b>) are the standard:
-                <div>
-                  Awesome frontend technologies such as <a href="http://facebook.github.io/react/" target="_blank">React</a> and <a href="http://angular.io" target="_blank">Angular</a> help to create them.<br />
-                  Thanks to <a href="http://redux.js.org/docs/recipes/ServerRendering.html" target="_blank">Universal Rendering</a>, they are now indexed by search engines.
-                </div>
+                <h3>Javascript webapps (<a href="https://en.wikipedia.org/wiki/Single-page_application" target="_blank">SPA</a>) are the standard.</h3>
```
Comment (dunglas) [inline src/containers/Home/Home.js:116]: Are those sections titles?

File: src/containers/Navigation/HeaderNav.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  render() {
+    const logoImage = require('./logo-header.svg');
+
+    return (
+      <header id="header">
+        <Navbar ref="navbar-container" inverse fixedTop={!this.state.inline}>
+          <Navbar.Header>
+            <Navbar.Toggle />
+            <Navbar.Brand>
+              <Link className="navbar-brand" to="/"><img height="48" src={logoImage} alt="API Platform" /></Link>
```
Comment (dunglas) [inline src/containers/Navigation/HeaderNav.js]: `height={48}`?

File: src/containers/Navigation/HeaderNav.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+            </Navbar.Brand>
+          </Navbar.Header>
+
+          <Navbar.Collapse>
+            <Nav>
+              <NavItem eventKey={1} href="https://api.github.com/repos/api-platform/api-platform/zipball" target="_blank"><span className="fa fa-download" /> Download</NavItem>
```
Comment (dunglas) [inline src/containers/Navigation/HeaderNav.js:72]: It should link to the download page and not directly to the ZIP.

File: src/theme/bootstrap.overrides.scss [hunk 7]
```diff
@@ -70,60 +92,85 @@ body {
   }
 }
 
-#incredible {
+.incredible {
```
Comment (dunglas) [inline src/theme/bootstrap.overrides.scss:110]: If we don't use them in CSS, maybe should we remove ids from the HTML too.

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by dunglas** [comment]
The new design is pretty! Thanks!!

I left some minor comments but it will be a great improvement to the website.

**Comment by dunglas** [comment]
Got it for the download link, the page has been removed. It makes sense.

There is a bug in the doc: the title is hided:

capture d ecran 2016-10-26 a 08 16 38

Also, links to anchors doesn't work (they send to the top of the page), but I think that this bug was already present before your PR.

There are also 2 new problems with Algolia. When the navbar is in the bottom, the search results are hided because they are displayed under the bar (not sure if there is an easy way to fix this).

capture d ecran 2016-10-26 a 08 19 10

Another new bug: when you go to the homepage then go to another page, Algolia doesn't work (it is working fine on the current version).

**Comment by dunglas** [comment]
It is possible to customize where to display Algolia's results, for instance it's what Batch is doing: https://batch.com/doc

File: src/containers/Navigation/HeaderNav.js (hunk 5 above)
Comment (Gregcop1) [reply src/containers/Navigation/HeaderNav.js]: why ?

File: src/containers/Home/Home.js (hunk 1 above)
Comment (Gregcop1) [reply src/containers/Home/Home.js]: Yep, bad conflict's resolve

File: src/containers/Home/Home.js (hunk 2 above)
Comment (Gregcop1) [reply src/containers/Home/Home.js:86]: There's a huge debate about that: having one h1 or multiple h1.
In term of SEO and accessibility, having a clear hierarchy is way better. The main info of your website is the title "API Platform" .
In term of accessibility, they can navigate inside of a page through a special menu built by heading level (and I don't think there's a reset on section for it)
The actual code doesn't built hierarchy at all. All infos are on the same level and I think we have to change this

File: src/containers/Navigation/HeaderNav.js (hunk 6 above)
Comment (Gregcop1) [reply src/containers/Navigation/HeaderNav.js:72]: I discuss that with @K-mos and this download page with two links is reaaalllly empty :)
IMHO, we should remove it until we have something to say in it

File: src/containers/Home/Home.js (hunk 2 above)
Comment (dunglas) [reply src/containers/Home/Home.js:86]: But we use HTML5, and the HTML5 outline algorithm is part of the spec: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines

IMO it's a urban legend that using the old way is better for SEO, and I generally prefer to respect the standard, but let's do this if you find that it's better.

File: src/theme/bootstrap.overrides.scss (hunk 7 above)
Comment (Gregcop1) [reply src/theme/bootstrap.overrides.scss:110]: Forget it... we can remove the whole CSS :) 
I don't know if you use those ids somewhere in JS but I can remove them if you want. But this PR shouldn't contain the whole revamp IMO cause there's really a lot of things to do to clean this mess

File: src/containers/Navigation/HeaderNav.js (hunk 5 above)
Comment (dunglas) [reply src/containers/Navigation/HeaderNav.js]: Because it's an int?

Comment (dunglas) [reply src/containers/Navigation/HeaderNav.js]: And for consistency (`eventKey={1}` for instance).

**Comment by Gregcop1** [comment]
Ok, I will check the Algolia's issue
At worst, I will tinker something in CSS

File: src/containers/Home/Home.js (hunk 2 above)
Comment (Gregcop1) [reply src/containers/Home/Home.js:86]: For SEO maybe but not for screen readers ;) And it's not out of the standard. The new standard don't remove the old way of writing structured heading

File: src/containers/Home/Home.js (hunk 4 above)
Comment (Gregcop1) [reply src/containers/Home/Home.js:116]: It's about structuring content importance. IMHO it's more important than a basic paragraph but I may be wrong

**Comment by Gregcop1** [comment]
I've just fix:
- [X] Algolia integration when the side is not fixed to top
- [X] Aloglia init script is now refresh each time a new header is build

**Comment by dunglas** [comment]
Thank you very much @Gregcop1

**Pull request merged**